karma-base.conf.js

```diff
@@ -13,7 +13,7 @@ module.exports = function (config) {
       'node_modules/systemjs/dist/system-polyfills.js',
       'node_modules/systemjs/dist/system.src.js',
       'node_modules/whatwg-fetch/fetch.js',
-      {pattern: 'node_modules/rxjs/bundles/Rx.js', watched: true, served: true, included: false},
+      {pattern: 'node_modules/rxjs/bundles/Rx.js', watched: true, served: true, included: true},
```

Comment:
It looks like we went from sytemJS loading the code to UMD. Why do we need to do this change? I think SystemJS was fine.